### PR TITLE
Update various locations to mention support for Ansible 2.8

### PR DESCRIPTION
**Pull request by sdodson**
/cc @kalexand-rh @mtnbikenc 
We added support for 2.8 here https://access.redhat.com/errata/RHBA-2019:2580

### CODE REVIEW

File: getting_started/install_openshift.adoc [hunk 1]
```diff
@@ -132,6 +132,11 @@ This command tells your RHEL system that the tools required to install
 {product-title} will be available from these repositories. Now we need the
 {product-title} installer that is based on Ansible.
 
+[NOTE]
+Older versions of {product-title} 3.11 supported only Ansible 2.6, the most
+recent versions of the playbooks have been updated to support Ansible 2.8 and
```
Comment (kalexand-rh) [inline getting_started/install_openshift.adoc]: If you're ok with this text, I'd open a follow-up to use this note instead:
> Older versions of {product-title} 3.11 supported only Ansible 2.6. The most
recent versions of the playbooks now support Ansible 2.8, which is the
preferred version to use.

**Comment by kalexand-rh** [comment]
@vikram-redhat, do you have a suggestion for getting this change approved this week?

File: getting_started/install_openshift.adoc (hunk 1 above)
Comment (sdodson) [reply getting_started/install_openshift.adoc]: that's fine with me

**Comment by vikram-redhat** [comment]
@mffiedler @sferich888 can you ack this?

**Comment by sdodson** [comment]
> @mffiedler @sferich888 can you ack this?
https://access.redhat.com/errata/RHBA-2019:2580 says
```
This release includes the following enhancements:

    Ansible 2.8+ is now supported. (BZ#1740842)
```

**Comment by jocolema** [comment]
I filed a bugzilla report to the Docs team to update the documentation on this.  Link below:

https://bugzilla.redhat.com/show_bug.cgi?id=1796629

**Comment by kalexand-rh** [comment]
@vikram-redhat got approval from @sferich888 through Slack, so I'm going to merge. Thank you all!

**Pull request merged**